datapacks/fleet/tiles/deep-water.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Regarding images, I will be adding a tool tip which shows the name in the near future so you may eventually want to remove the names